ethdk/src/Bls/BlsNetworks.ts

```diff
@@ -9,13 +10,13 @@ const localhost: BlsNetwork = {
   verificationGateway: '0x689A095B4507Bfa302eef8551F90fB322B3451c6',
 }
 
-const NETWORKS = {
-  localhost,
-}
-
-export function getNetwork(name?: string): BlsNetwork {
-  if (name === 'localhost' || name === undefined) {
-    return NETWORKS.localhost
+export function getNetwork(network?: Network): BlsNetwork {
+  if (network === undefined) {
+    // Return default network
+    return localhost
+  }
+  if (network?.type === 'bls') {
```

Comment:
Add this for a typeguard check, this function will make sure the network that is returned is typed as BlsNetwork